README.md

```diff
@@ -141,4 +142,4 @@ See docs/Testing.rst.
 
 ## Contributing to Swift
 
-Contributions to Swift are welcomed and encouraged! Please see the [Contributing to Swift guide](https://swift.org/contributing/).
```

Comment:
I have not changed here. I do not know why it I got conflict. Done the changes. Thanks